x/gov/types/keys.go

```diff
@@ -52,11 +52,21 @@ var (
 
 var lenTime = len(sdk.FormatTimeBytes(time.Now()))
 
+// GetProposalIDBytes returns the byte representation of the proposalID
+func GetProposalIDBytes(proposalID uint64) (proposalIDBz []byte) {
```

Comment:
This now makes it state machine breaking, correct?

Reply:
Yes